rosdep/base.yaml

```diff
@@ -4545,6 +4545,16 @@ muparser:
   fedora: [muParser-devel]
   gentoo: [dev-cpp/muParser]
   ubuntu: [libmuparser-dev]
+nanoflann:
+  arch: [nanoflann]
+  debian:
+    buster: null
+    '*': [libnanoflann-dev]
```

Comment:
CI is complaining because the '*' should be before other items.  Same for ubuntu below.